jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -179,6 +155,10 @@ public void buildDocker()
     } catch (InvalidImageReferenceException ex) {
       throw new GradleException(
           HelpfulSuggestions.forInvalidImageReference(ex.getInvalidReference()), ex);
+
+    } finally {
+      // TODO: This should not be called on projectProperties.
```

Comment:
I think these TODOs are a bit redundant since there's already one in `ProjectProperties`.

Reply:
This is about #1358. I can make the comment clearer.

Reply:
Right, I meant there's already a todo about moving these methods out of `ProjectProperties`, inside the interface itself. I don't think we need extra todos where they're called as well.